_data/events.yml

```diff
@@ -1,13 +1,13 @@
-  - date: 2015-01-16
-    name: Data Day Texas
-    info: "Open Source Lambda Architecture with Kafka, Samza, Hadoop, and Druid"
-    link: http://datadaytexas.com/
-
   - date: 2015-01-28
     name: Los Angeles High Scalability Meetup
     info: "Real Time Analytics at Scale with Druid"
     link: http://www.meetup.com/scalela/events/227559808/
 
+  - date: 2015-02-25
+    name: Druid Feburary Meetup
```

Comment:
typo, also do you mind sending a message to the user list to see who might want to host it / who would like to speak?